scripts/ci/k3d-cluster.sh

```diff
@@ -5,7 +5,7 @@ set -o pipefail
 
 # CLUSTER_NAME is fuse-( branch, or tag, fallback to ref )
 CLUSTER_SUFIX=$(echo $(git symbolic-ref -q --short HEAD 2>/dev/null || git describe --tags --exact-match --short HEAD 2>/dev/null || git describe --all | tr '/' -) | tr _ - )
-CLUSTER_NAME=fuseml-${CLUSTER_SUFIX}
+CLUSTER_NAME=fuseml-${CLUSTER_SUFIX//[!A-Za-z0-9-]/-}
```

Comment:
Why not just remove the problematic characters?

Reply:
I can do that too, sure, but it's essentially the same result, so I don't really see a reason to.